D/diffutils/build_tarballs.jl

```diff
@@ -0,0 +1,41 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "diffutils"
+version = v"3.7.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource(
+        "https://ftp.gnu.org/gnu/diffutils/diffutils-3.7.tar.xz",
+        "b3a7a6221c3dc916085f0d205abf6b8e1ba443d4dd965118da364a1dc1cb3a26",
+    ),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/diffutils-*/
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make -j$(nproc)
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = Product[
+    ExecutableProduct("cmp", :_cmp)
+    ExecutableProduct("diff", :_diff)
```

Comment:
These names are already defined in Base, unfortunately, so BinaryBuilder doesn't allow us to use them. See https://github.com/JuliaPackaging/BinaryBuilderBase.jl/issues/55